tasks/config_manager/load.yaml

```diff
@@ -4,15 +4,13 @@
 - name: initialize function
   include_tasks: includes/init.yaml
 
-- name: validate task arguments
-  validate_role_spec:
-    spec: load_config_spec.yaml
+- name: validate config_manager_text is defined
+  fail:
+    msg: "missing required arg: config_manager_text"
+  when: config_manager_text is undefined
 
-- name: lookup config file
-  set_fact:
-    iosxr_config_text: "{{ lookup('config_template', iosxr_config_file) | join('\n') }}"
-  when: iosxr_config_file != ''
-
-- name: Include configure task
-  include_tasks: includes/configure/configure.yaml
-  when: not ansible_check_mode and not iosxr_config_replace
+- name: load configuration into device
```

Comment:
@justjais replace functionality is missing from this task? You may refer to https://github.com/ansible-network/vyos/blob/devel/tasks/config_manager/load.yaml#L21